Makefile

```diff
@@ -21,6 +21,10 @@ ifeq ($(TEST_FLAGS),)
 	TEST_FLAGS := -cover
 endif
 
+ifeq ($(BUILD_TAGS),)
```

Comment:
Is the comma added at the end is intentional?

Reply:
Yes, it is to check if the env variable is not set